src/WebServer/Http/HttpServer.cs

```diff
@@ -19,9 +20,11 @@ public class HttpServer : IDisposable
         private readonly StreamSocketListener _listener;
         private readonly SortedSet<RouteRegistration> _routes;
         private readonly ContentEncoderFactory _contentEncoderFactory;
+        private ILogger _log;
 
         public HttpServer(int serverPort)
         {
+            _log = LogManager.GetLogger<HttpServer>();

```

Comment:
My opinion was that you wouldn't set this on an interface level, but on the underlying implementation. 
See https://github.com/tomkuijsten/restup/pull/76/files#diff-5ef3cb421074b340731c3282f5018f5bR26 for how to set your own implementation.